model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/entities/ClientScopeListQuery.java

```diff
@@ -25,6 +25,7 @@
     private final Set<String> clientScopes;
     private final String realm;
     private final String realmName;
+    private String clientId;
```

Comment:
Is this `clientId` or `clientUuid`?

Reply:
`clientUuid`, I'll rename it to avoid confusion.